server/app/lib/config.py

```diff
@@ -42,6 +42,24 @@
     "apachecon": "apachecon.com",
 }
 
+# These projects don't use Jira
+# TODO: derive this list dynamically somehow
+NON_JIRA_PROJECTS = [
+  "accumulo",
+  "annotator",
+  "celix",
+  "cordova",
+  "couchdb",
+  "fluo",
+  "kibble",
+  "linkis",
+  "lucene",
+  "netbeans",
+  "skywalking",
+  "trafficcontrol",
+  "trafficserver",
```

Comment:
What about the Bugzilla ones ?
Tomcat and HTTPD also need to reject requests often.